extra-python/simplemediawiki/autobuild/beyond

```diff
@@ -0,0 +1,2 @@
+python setup.py install --prefix=/usr --root="$PKGDIR" --optimize=1
+install -Dm644 COPYING "$PKGDIR"/usr/share/licenses/"$PKGNAME"/LICENSE
```

Comment:
Move this file to `autobuild/build`

Reply:
> Sorry, didn't entirely understand your intention.
> 
> `autobuild` supports building Python package, is this package only supports `python-3` ?

no, I see this pkg have python2 file, I should PKGDEP add `python-2`?

Reply:
> Sorry, didn't entirely understand your intention.
> 
> `autobuild` supports building Python package, is this package only supports `python-3` ?

```root@saki-make_111ea5 [ tree@testing ] # dpkg -L python3-simplemediawiki | grep python2
/usr/lib/python2.7
/usr/lib/python2.7/site-packages
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info/PKG-INFO
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info/SOURCES.txt
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info/dependency_links.txt
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info/requires.txt
/usr/lib/python2.7/site-packages/simplemediawiki-1.2.0b2-py2.7.egg-info/top_level.txt
/usr/lib/python2.7/site-packages/simplemediawiki.py
/usr/lib/python2.7/site-packages/simplemediawiki.pyc
/usr/lib/python2.7/site-packages/simplemediawiki.pyo
```

Reply:
In this case, you need to test if the Python 2 version actually works or if you need for sure it will work properly. If it compatible with Python 2, then just remove `autobuild/build`